tox.ini

```diff
@@ -12,34 +12,35 @@ envdir =
     pypy3: {toxworkdir}/pypy3
     py: {toxworkdir}/py
 commands =
-    python housekeep.py setup
-    nocov: python -m nose2 -v {posargs}
-    cov: python -m coverage run -m nose2
-    cov: python -m coverage combine
-    cov: python -m coverage html --directory=htmlcov/{envname}
-    cov: python -m coverage report -m --fail-under=100
-    cov: python -m coverage xml
-    diffcov: diff-cover coverage.xml --html-report diffcov.html
-    diffcov: diff-cover coverage.xml --fail-under=100
-    profile: python -m nose2 --profile {posargs}
+    python housekeep.py prep
+    nocov: pytest --verbose -p no:cov {posargs}
+    cov: pytest --cov --cov-report=xml --cov-report=html --cov-report=term {posargs}
+    diffcov: diff-cover coverage-{env:INTERP}.xml --html-report diffcov-{env:INTERP}.html
+    diffcov: diff-cover coverage-{env:INTERP}.xml --fail-under=100
+    profile: pytest --profile {posargs}
     python housekeep.py gather
 #sitepackages = True
 usedevelop = True
 deps =
-    nose2
-    coverage
+    colorama
+    coverage[toml]
+    packaging
+    pytest
+    pytest-cov
+    pytest-mock
+    pytest-profiling
+    pytest-sugar
     diff_cover
 setenv =
-    cov: COVERAGE_RCFILE=coverage.cfg
-    cov: COVERAGE_PROCESS_START=coverage.cfg
-    cov: COVERAGE_OPTIONS="-p"
```

Comment:
Removed because:

* Configuration is now inside `pyproject.toml`, which will automatically be scanned by `coverage[toml]` (see deps above)
* We're not using subprocesses anyways so no point in setting `COVERAGE_PROCESS_START`
* The `-p` option is equivalent to `parallel = true` in `pyproject.toml`